### ISSUE TITLE
Request to merge in new website

### ISSUE THREAD

**Issue by Songtech-0912** [NatronGitHub/Natron#1013]
After 3 years of development, the new website is done! A huge shout-out to @Shrinks99 and Siddhant for making this possible! A preview can be seen [here](https://songtech-0912.github.io/Natron-redesigned-website/).

In terms of technical information:

- The site is a modern Jekyll site that comes with GitHub actions already, basically all batteries-included
- The website is **completely content-complete** and ready for launch
  - It also has release notes ready on its news page
  - And all the installers/packages/binaries we build in one place, the links to github releases are templated instead of hardcoded so that will be a lot less work going forwards
  - It provides a good foundation for us to add e.g. article tutorials because all the content is markdown and easy to add new content to, especially in something like [Obsidian](https://obsidian.md)
- A few concerns raised were:
  - Licensing issues (we have all assets embedded in the website, with their licenses, but it makes licensing annoying and privacy issues), which we wanted a package manager for, but the overhead was just too much and we couldn't make it work

So I think everything is ready for launch! The process for launch, as I understand, would be something like this:

- Have a final review before the go-ahead (although imo the website should be good as-is)
- Rename and archive the existing natron website repository, maybe to a name like `natron-legacy-website`
- Transfer ownership of the repo from Hank’s GitHub over to Natron’s official Github

I hope we can get this finally out there! @devernay @bonalex01dev I would love if you could take a look at this, and if things look okay, give the go-ahead.

**Comment by devernay** [comment]
Can you tell us a bit more what actions have to be done when a new release comes out? 

 The current release (2.5) doesn't work on Apple SIlicon Macs, so I should have a different download link for Apple SIlicon Macs that points to the 2.6.0 alpha. How easy is it to do? The 2.5 download should be "macOS (Intel only)", and there should be a link to the prerelease (maybe just the GH page is ok) for Apple SIlicon

**Comment by Songtech-0912** [comment]
Yes, certainly! The process is fairly straightforward. First, I forgot to mention, (my copy of) the new website repository is [here](https://github.com/Songtech-0912/Natron-redesigned-website/tree/main), Hank's is equivalent. 

1. There is a `_config.yml` in the root folder, where, so long as our naming conventions for the CI-built binaries remain the same, you literally just need to change the values of 3 keys (takes about 15 seconds to do)
2. In the `site_collections/_posts` folder, that's where the release news is, create a new markdown file with the release info. But copying the changelog over is basically enough, and we'd have a changelog for releases anyways.
3. In `templates/releasegraphic` Hank wrote a really nice Python script that you can call with a single commandline to automatically generate a release graphic.
4. That's all! Push and the release is done.

That's basically it! @Shrinks99 correct me if I'm missing some stuff but I think this should be it. As for adding a download link, that is quite quick actually. I may be able to do it tonight after schoolwork.

**Comment by Shrinks99** [comment]
Yeah adding a download link itself is pretty easy, making it work nicely with GitHub releases and variables requires slightly more thought (naming conventions of builds).  I'd just do it as a one-off and then update along with 2.6.0.

**Comment by Songtech-0912** [comment]
@Shrinks99 I made a PR with the changes Fred suggested as well as detailed information to update the site for each new release, see https://github.com/Shrinks99/NatronGitHub.github.io/pull/14

**Comment by Shrinks99** [comment]
Merged!  @devernay I believe all of your above requests have been handled in that PR :)
[
Detailed release instructions can be found in the readme file](https://github.com/Shrinks99/NatronGitHub.github.io/tree/main?tab=readme-ov-file#authoring-an-update-post-for-new-releases).

**Comment by Shrinks99** [comment]
@devernay One point that I do need to draw your attention to is your name on the previous release posts.  I pulled a bunch of the information from the past GitHub releases, but it would be uncool of me to not make that explicitly clear.  If you want I can replace the author tag with "Natron Maintainers" or similar?

Other than that, this is ready to go and I would really like to see it out in the world!  I'm happy to hop on a call if you would like a more in-depth walkthrough of the site / want to transfer things and push it live together?

**Comment by Songtech-0912** [comment]
@devernay Is it possible to get it launched now? AFAIK we've ironed out the kinks and basically everything should be ready for launch (so to speak). The steps are the same as previously-outlined, but I'll repeat here:

- Have a final review before the go-ahead (although with all our fixes the website should be good as-is)
- Rename and archive the existing natron website repository, maybe to a name like `natron-legacy-website`
- Transfer ownership of the repo from Hank’s GitHub over to Natron’s official Github and deploy on github pages (this is something that you can do straight from GitHub UI)

The whole process would probably just take a day or two while the benefits can be huge:

1. It will reassure people that Natron is not dead (seriously the amount of "is Natron dead" questions we get is ridiculous) because it is active development
2. It gives a modern face to the project, because the website, after all, is an important part of our branding
3. It makes it Natron more appealing to a wider audience (because honestly people _will_ judge software based on appearance and a pretty landing page establishes a good first impression). 

I can understand if you don't have time for this right now, which is completely fine. I just hope at some point we can get this done.

> P.S. It might be helpful to coordinate this on the Natron discord server if you have a discord, since me, Hank, and Siddhant are all on it and most of our development discussions are on there.

**Comment by devernay** [comment]
@acolwell @YakoYakoYokuYoku do you have time to review this and approve? Tell me if you need additional permissions

**Comment by acolwell** [comment]
@devernay I don't have a ton of time to work on this, but I am happy to review this and facilitate its deployment. 

Based on @Songtech-0912 comment above about the necessary steps to deploy and https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization I suspect I'll need the "All-repository admin" role at the organization level to help them deploy this. I understand this role comes with a great amount of power and I hope that given my work over the last 2 years you are willing to trust me with it. I'm also happy to commit to downgrading back to a lower permission role once this is complete if you'd like.

@Shrinks99 and @Songtech-0912 I'll try to do an initial review pass over your repository sometime this weekend. Assuming I don't find any major issues we can coordinate on Discord when to do the actual transfer to the new repository.

**Comment by Shrinks99** [comment]
@acolwell Sounds good.  I'm somewhat busy this weekend but happy to transfer ownership over to the Natron org whenever!  DM me on Discord if you need to reach me.

**Comment by YakoYakoYokuYoku** [comment]
I approve the new design, submit a PR to NatronGitHub/NatronGitHub.github.io for me and any other interested in to review it. Might be out of scope, maybe another issue, but could a similar color palette be applied to the generated documentation?

**Comment by Songtech-0912** [comment]
@YakoYakoYokuYoku @acolwell Thanks so much! Hank's idea I believe is to do a direct transfer of the repository from his account since the new website is basically a complete rewrite and shares 0% code with the existing website, so a PR may not be most appropriate. I can open an _issue_ on NatronGitHub/NatronGitHub.github.io though, if you would like that?

Also - about the docs, I looked into this a while ago, and did a little test using the [Furo](https://pradyunsg.me/furo/) Sphinx theme (it's been pretty popular in the Python scientific community?) which was pretty easy to set up, and it offers [color customization options](https://pradyunsg.me/furo/customisation/colors/). So I don't think it would be that much work, if you guys would like me to do a (separate) PR for that, I'd be happy to!

![Image](https://github.com/user-attachments/assets/b22ef0c6-d141-4329-88c4-558241fe00f1)

**Comment by Shrinks99** [comment]
@YakoYakoYokuYoku For [best GitHub Pages compatibility](https://docs.github.com/en/pages/getting-started-with-github-pages/what-is-github-pages#types-of-github-pages-sites), the old repo should be renamed `NatronGitHub.github.io-v1` or similar and archived.  The new site should be transferred over with the existing name.

Perhaps we could set this up on a call together?  Should be a 5 minute task.  It involves a little bit of coordination because of the naming and transferring :)

**Comment by YakoYakoYokuYoku** [comment]
I've got another idea, making a branch under the oldest commits of that repo. But maybe what you say might suffice.

**Comment by acolwell** [comment]
I agree with @Shrinks99 and think the repo transfer is the cleanest solution. I'm just waiting for @devernay to respond and give me the necessary permissions to help facilitate that process.

**Comment by Songtech-0912** [comment]
Glad to see the hard work you've been putting into this (reviewing and preparing for launch) guys! You're the best, I am excited for the launch!

**Comment by devernay** [comment]
@acolwell I think you're good to go!

**Comment by Songtech-0912** [comment]
So glad to hear! Me and Hank are on the Natron discord so @acolwell let us know there if there's anything you need us to do.

**Comment by rodlie** [comment]
The new website is live: https://natrongithub.github.io

OG image is missing (need a fallback). And docs should open in new tab (imho).

Thanks for the work @Shrinks99 and @Songtech-0912

**Comment by Songtech-0912** [comment]
Those should be a quick fix, we're working on that right now, but I think it should be okay to close this issue now?

**Comment by rodlie** [comment]
Ok, just do a PR and I will review.

Also, fix whitespace in desc:
```
  <meta name="og:description" content="
      Open Source VFX Compositing Software
    ">
```

You are also missing generic `description` meta.
